web-animations/interfaces/Animatable/getAnimations.html

```diff
@@ -91,13 +93,15 @@
   const div = createDiv(t);
   const animation = div.animate(null, 100 * MS_PER_SEC);
 
-  animation.effect.timing.endDelay = -200 * MS_PER_SEC;
+  animation.effect.updateTiming({
+    endDelay: animation.effect.getTiming().endDelay - 200 * MS_PER_SEC,
```

Comment:
I don't follow this change. The previous version just set it to `-200 * MS_PER_SEC`, but this version decrements the existing value by `200 * MS_PER_SEC` - why ?

Reply:
That's a bug in the patch. Thanks!